src/west/main.py

```diff
@@ -557,12 +557,10 @@ def main(argv=None):
     except KeyboardInterrupt:
         sys.exit(0)
     except CalledProcessError as cpe:
-        log.err('command exited with status {}: {}'.format(
-            cpe.args[0], quote_sh_list(cpe.args[1])))
```

Comment:
I can guarantee this worked, I tested it explicitly. That said, this is the right way to do this.

Reply:
I'm confused. This change is what we discussed in previous PRs, so that users who initialize CalledProcessError with kwargs instead of positionals don't get faced with an IndexError when we try to access `cpe.args`.

Reply:
IOW: I know you're not asking for a change here, but you specifically asked for this change earlier :). Even though it "works" if you properly initialize CalledProcessError as the python subprocess module does, users who make their own instances won't always, as we discovered.

Reply:
The change is required and I am completely happy with it. I am just saying that this old code actually worked, to my amazement :)